app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<% content_for(:add_item_button) { link_to 'Add Post', new_admin_post_path, class: 'btn btn-primary' } %>
+
+<div class="row py-3">
+  <h2><%= @title %></h2>
+  <div class="col-12">
+    <% @posts.each do |post| %>
+      <%= render partial: 'admin/shared/post', locals: {post: post} %>
+      <% end unless @posts.blank? %>
```

Comment:
https://github.com/tux255/sports_hub_portal/pull/7#discussion_r658530976

Reply:
it gives error if @categories are empty

Reply:
what kind of error?